charts/csi-secret-provider-class/values.yaml

```
@@ -14,3 +14,36 @@ secretObjects: []
 # secretObjectsTpl -- template to generate secretObjects
 # @default -- raw values from `secretObjects`
 secretObjectsTpl: "{{ .Values.secretObjects | toYaml }}"
+
+# Creates a dummy Deployment which uses the csiVolume. This is a hack, in case your Chart doesn't support defining extraVolumes
+# DISCLAIMER: DO NOT USE THIS unless as last resort.
+# Always try to use extraVolume Bindings inside your already existing Chart or application to make csi-secret-store manage the secret. And if there aren't any possibilities to mount additional volumes in your chart, try and create a pull-request upstream adding the feature to the chart.
+dummyDeployment:
+  # dummyDeployment.enabled -- enable the dummy deployment
+  # @default -- false
+  enabled: false
+  image:
+    # dummyDeployment.image.repository -- Container image to deploy
+    # @default -- k8s.gcr.io/pause
+    repository: k8s.gcr.io/pause
+    # dummyDeployment.image.tag -- sets the image tag to use
+    # @default -- 3.4.1
+    tag: 3.4.1
+    # dummyDeployment.image.pullPolicy -- When to pull the container image
+    # @default -- ifNotPresent
+    pullPolicy: ifNotPresent
```

```suggestion
    pullPolicy: IfNotPresent
```
Also, don't duplicate `@default` if it matches the actual value!